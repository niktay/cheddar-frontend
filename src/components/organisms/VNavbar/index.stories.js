import { storiesOf } from "@storybook/vue";
import VNavbar from ".";

storiesOf("Organism - VNavbar", module).add("default", () => ({
  components: { VNavbar },

  template: `<VNavbar />`
}));
