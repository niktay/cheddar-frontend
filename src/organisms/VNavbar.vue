<template>
  <div class="o-VNavbar">
      <VLogo />
      <a href="#" class="login-button"><i class="fas fa-user-circle"></i></a>
  </div>
</template>

<script>
import VLogo from "@/atoms/VLogo";

export default {
  components: {
    VLogo
  }
};
</script>

<style scoped lang="scss">
.o-VNavbar {
  grid-column: 2/3;
  grid-row: 1/1;
  margin-top: 2rem;
  padding-bottom: 0.5rem;

  .login-button {
    font-size: 2rem;
    float: right;
    color: #0d0d0d;
    &:hover {
      color: #5d5d5d;
    }
    &:active {
      color: #020202;
    }
  }
}
</style>
