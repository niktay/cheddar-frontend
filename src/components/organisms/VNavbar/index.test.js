import { mount } from "@vue/test-utils";
import VNavbar from ".";

describe("VNavbar component", () => {
  it("should render correctly", () => {
    const comp = mount(VNavbar);

    expect(comp.html()).toMatchSnapshot();
  });
});
