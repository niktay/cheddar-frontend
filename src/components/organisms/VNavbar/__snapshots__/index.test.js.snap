// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VNavbar component should render correctly 1`] = `<nav class="o-VNavbar"><img src="@/assets/logo.svg" alt="Cheddar Logo" class="a-VLogo"> <a href="#" class="login-button"><i class="fas fa-user-circle"></i></a></nav>`;
